### PR TITLE
WIP: dev-java/openjdk-bin: Latest jdk release jdk15

### PR DESCRIPTION
**Pull request by dblaber**
Adding ebuild for the latest non-lts jdk release, jdk15.

Signed-off-by: Darren Blaber dmbtech@gmail.com

### CODE REVIEW

File: dev-java/openjdk-bin/openjdk-bin-15_p36.ebuild [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		rm -v lib/src.zip || die
+	fi
+
+	rm -v lib/security/cacerts || die
+	dosym ../../../../etc/ssl/certs/java/cacerts \
+		"${dest}"/lib/security/cacerts
+
+	dodir "${dest}"
+	cp -pPR * "${ddest}" || die
+
```
Comment (gyakovlev) [inline dev-java/openjdk-bin/openjdk-bin-15_p36.ebuild:87]: check for `::gentoo` version

I have a stable symlink here

```
# provide stable symlink
    dosym "${P}" "/opt/${PN}-${SLOT}"
```

File: dev-java/openjdk-bin/openjdk-bin-15_p36.ebuild [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	$(abi_uri aarch64 arm64)
+	$(abi_uri ppc64le ppc64)
+	$(abi_uri x64 amd64)
+"
+
+DESCRIPTION="Prebuilt Java JDK binaries provided by AdoptOpenJDK. Short Term Supported"
+HOMEPAGE="https://adoptopenjdk.net"
+LICENSE="GPL-2-with-classpath-exception"
+KEYWORDS="~amd64 ~arm64"
+IUSE="alsa cups -gentoo-vm headless-awt source"
```
Comment (gyakovlev) [inline dev-java/openjdk-bin/openjdk-bin-15_p36.ebuild:29]: replace the `-` with `+` , it will be masked, but enabled.

**Review by gyakovlev — CHANGES_REQUESTED** [review]
minor nits.
otherwise works =) but I'll test it more a bit later.

**Comment by ConiKost** [comment]
Since we have now jdk17, I would close this, as version 12-16 are non lts, so I don't think we need them anymore.

**Pull request closed without merging**